python/veles/compatibility/pep487.py

```diff
@@ -42,7 +42,7 @@ def __new__(cls, *args, **kwargs):
 
             return self
 
-    class NewObject(metaclass=NewType):
+    class NewObject(object):
```

Comment:
Czemu nie używasz with_metaclass tutaj, tylko zrzucasz to na klasę dziedziczącą?